Lib/fontbakery/profiles/googlefonts_conditions.py

```diff
@@ -540,3 +540,13 @@ def gfonts_repo_structure(fonts):
   #        about the expected structure.
   abspath = get_absolute_path(fonts[0])
   return abspath.split(os.path.sep)[-3] in ["ufl", "ofl", "apache"]
+
+
+@condition
+def production_metadata():
+  """Get the Google Fonts production metadata"""
+  import json
+  import requests
+  meta_url = "http://fonts.google.com/metadata/fonts"
+  # can't do requests.get("url").json() since request text starts with ")]}'"
```

Comment:
These 4 spurious chars in the begining of the json file are super weird! I'm afraid this code will break soon...

![Screenshot from 2020-07-21 09-57-03](https://user-images.githubusercontent.com/213676/88057589-a9775a80-cb38-11ea-8323-d8c5d917e6db.png)

Reply:
but if it ever breaks, it will be detected by the code-tests included in this PR, as they invoke this condition. So, I think we're safe.